bylaws.md

```diff
@@ -94,6 +93,6 @@
 1. A member in Good Standing is defined as someone who:
 	- Has paid dues on time as defined by the denhac Membership Agreement.
 	- Is not actively subject to disciplinary action. 
-2. A Simple majority vote is a vote where more than 50% of the board members must vote in favor of the motion. The Chair does not cast a vote except in the case of a tie
+2. A Simple majority vote is a vote where more than 50% of all board members must vote in favor of the motion.
```

Comment:
`A Simple majority vote is a vote where more than 50% of the board members must vote in favor of the motion.`
There was recent confusion with board members absent from a meeting and if `50% of the board members` meant the entire board or those currently present. 

Changed language to "50% of **all** board members" to clarify it requires a majority of all board members and not just those present. 
Note - Intentionally didn't specify `at least four (4) out of the seven (7)` like rules for super majority for instances of vacancies.